package-lock.json

```diff
@@ -1,10208 +0,0 @@
-{
```

Comment:
Why delete this one? Won't this cause repository-level dependencies to not be locked anymore?

Reply:
Good catch! reverting!